package.json

```diff
@@ -118,22 +112,20 @@
     "react-dom": "^18.2.0",
     "react-is": "^18.2.0",
     "rimraf": "^5.0.5",
-    "sanity": "^3.21.3",
-    "sanity-extra": "^1.0.0",
-    "sanity-testing-library": "^1.0.0",
-    "semantic-release": "^22.0.0",
+    "sanity": "^3.67.1",
```

Comment:
It never occurred to me to just do this. So simple, yet so effective.
devDep sanity and a selfcontained sanity.config.ts– so smooth 👍 
_This_ is the way plugin-kit should ship.